jax/_src/compute_on.py

```diff
@@ -40,10 +40,11 @@ def current_compute_type() -> str | None:
   return config.compute_on_context_manager.value
 
 def _check_valid(c_type: str):
-  if c_type not in {'device_host', 'device', 'tpu_sparsecore'}:
+  if (c_type not in {'device_host', 'device', 'tpu_sparsecore'}
+      and not c_type.startswith("gpu_stream:")):
     raise ValueError(
         f'Invalid compute type {c_type}. Current supported values '
-        'are `device_host`, `device` and `tpu_sparsecore`.')
+        'are `device_host`, `device`, `tpu_sparsecore`, and `gpu_stream:#`. ')
```

Comment:
nit:

```suggestion
        'are `device_host`, `device`, `tpu_sparsecore`, and `gpu_stream:#`.')
```

Reply:
Let's fix these as a follow up since the merge process already started. Great catch BTW!